test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6231,6 +6231,21 @@ def forward(self, input):
         self.run_test(DimModel(), empty_input)
         self.run_test(DimModel(), multi_dim_input)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_dim_1(self):
+        class M(torch.jit.ScriptModule):
+            @torch.jit.script_method
```

Comment:
Do we need this for forward() method?

Reply:
See https://brsoff.github.io/tutorials/advanced/cpp_export.html
"we can convert it to a ScriptModule by subclassing it from torch.jit.ScriptModule and adding a @torch.jit.script_method annotation to the model’s forward method:"

Reply:
I checked JIT document here https://pytorch.org/docs/stable/jit.html#migrating-to-pytorch-1-2-recursive-scripting-api, and for forward() method, it should be the default behavior so we don't need it anymore.

Anyway, it doesn't hurt by adding this.

Reply:
`@torch.jit.script_method` on forward is required for classes inherited from `torch.jit.ScriptModule`. That being said, it is indeed deprecated, and the preferred way is to inherit from `torch.nn.Module`, and call `torch.jit.script(M())`.